hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OMConfigKeys.java

```diff
@@ -270,6 +270,13 @@ private OMConfigKeys() {
 
   public static final String OZONE_OM_METADATA_LAYOUT_PREFIX = "PREFIX";
 
+  // Defines the default bucket behaviour when OM does not pass the
+  // bucket layout argument during bucket creation.
+  public static final String OZONE_DEFAULT_BUCKET_LAYOUT =
+      "ozone.default.bucket.layout";
+  public static final String OZONE_DEFAULT_BUCKET_LAYOUT_DEFAULT = "OBS";
```

Comment:
Like my earlier comment, please use full name OBJECT_STORE instead of OBS and FILE_SYSTEM_OPTIMIZED instead of OBS.